routes/forums.js

```diff
@@ -144,12 +144,29 @@ router.get('/:forum_id/', (req, res) => {
 
 // Update specific fields of forum
 router.put('/:forum_id', (req, res) => {
-
+    // TODO Confirm user, confirm changes
+    let forumId = req.params.forum_id;
```

Comment:
I can pass empty string for title and content and it's updated. Do we want that?

Reply:
If an empty string is passed, it will now just ignore that field. Will still update any other changes.